src/mousemanager.h

```diff
@@ -41,31 +37,20 @@ class MouseManager : public Object {
     void grab_client_buttons(Client* client, bool focused);
 
     void mouse_handle_event(unsigned int modifiers, unsigned int button, Window window);
-    void mouse_initiate_drag(Client* client, MouseDragFunction function);
     void mouse_stop_drag();
     bool mouse_is_dragging();
     void handle_motion_event(Point2D newCursorPos);
 
+    void mouse_initiate_drag(Client* client, void (MouseDragHandler::*function)(Point2D));
     void mouse_initiate_move(Client* client, const std::vector<std::string> &cmd);
     void mouse_initiate_zoom(Client* client, const std::vector<std::string> &cmd);
     void mouse_initiate_resize(Client* client, const std::vector<std::string> &cmd);
     void mouse_call_command(Client* client, const std::vector<std::string> &cmd);
-    /* some mouse drag functions */
-    void mouse_function_move(Point2D newCursorPos);
-    void mouse_function_resize(Point2D newCursorPos);
-    void mouse_function_zoom(Point2D newCursorPos);
 
 private:
     //! check whether we can continue dragging
```

Comment:
Is this comment still appropriate? Maybe just leave it out

Reply:
Ah you somehow made this class represent the boolean of dragging possibility? I guess I have to take closer look first...

Reply:
This comment is obsolete now! Sorry for the confusion.

The MouseDragHandler class now also takes care of this boolean of dragging possibility. But this class does more than that, namely the actual window geometry changes during the drag.